src/code/z_demo.c

```diff
@@ -391,176 +443,189 @@ void func_80064824(PlayState* play, CutsceneContext* csCtx, CsCmdBase* cmd) {
                 }
             }
             break;
-        case 28:
-            play->unk_11DE9 = true;
+
+        case CS_MISC_HALT_ALL_ACTORS:
+            play->haltAllActors = true;
             break;
-        case 29:
-            play->unk_11DE9 = false;
+
+        case CS_MISC_RESUME_ALL_ACTORS:
+            play->haltAllActors = false;
             break;
-        case 30:
-            Flags_SetEnv(play, 3);
+
+        case CS_MISC_SET_FLAG_3:
+            CutsceneFlags_Set(play, 3);
             break;
-        case 31:
-            Flags_SetEnv(play, 4);
+
+        case CS_MISC_SET_FLAG_4:
+            CutsceneFlags_Set(play, 4);
             break;
-        case 32:
-            if (sp3F != 0) {
+
+        case CS_MISC_SANDSTORM:
+            if (isFirstFrame) {
                 play->envCtx.sandstormState = SANDSTORM_FILL;
             }
+
             func_800788CC(NA_SE_EV_SAND_STORM - SFX_FLAG);
             break;
-        case 33:
+
+        case CS_MISC_SUNSSONG_START:
             gSaveContext.sunsSongState = SUNSSONG_START;
             break;
-        case 34:
+
+        case CS_MISC_FREEZE_TIME:
             if (IS_DAY) {
                 gSaveContext.dayTime -= gTimeSpeed;
             } else {
                 gSaveContext.dayTime -= gTimeSpeed * 2;
             }
             break;
-        case 35:
-            AudioOcarina_PlayLongScarecrowAfterCredits();
-            csCtx->frames = cmd->startFrame - 1;
+
+        case CS_MISC_LONG_SCARECROW_SONG:
+            AudioOcarina_PlayLongScarecrowSong();
+            csCtx->curFrame = cmd->startFrame - 1;
```

Comment:
does curFrame = startFrame-1 mean the cutscene never ends and `AudioOcarina_PlayLongScarecrowSong` is called every frame?

Reply:
yea, the command will just go on forever even after the song is done.
text command has the same mechianic, but its not forever. it will wait till text has been advanced to continue

Reply:
It may be nice to have a comment pointing this out, as it's a rare/unique occurrence?